src/pages/FollowList/components/FollowList.tsx

```diff
@@ -0,0 +1,65 @@
+import { useQuery } from 'react-query';
+import { getMe } from 'api/user';
+import { ReactComponent as Arrow } from 'assets/svg/common/arrow.svg';
+import cn from 'utils/ts/classNames';
+import useBooleanState from 'utils/hooks/useBooleanState';
+import { FollowListInfo } from '../static/entity';
+import Follower from './Follower';
+import style from './FollowList.module.scss';
+
+// 유저 검색 시 자신의 정보는 안보이도록 자신의 정보를 받아서 비교, 자신과의 followedType은 NONE
+const useGetMe = () => {
+  const { data } = useQuery('me', getMe);
+
+  return { data };
+};
+
+export default function FollowList({ title, data, user }: FollowListInfo) {
+  const me = useGetMe();
+  const [value, , ,toggle] = useBooleanState(true);
+
+  return (
+    <div className={style.container}>
+      <button onClick={toggle} type="button" className={style.title}>
+        {title}
+        <Arrow className={cn(
+          {
+            [style['title__arrow--up']]: value,
+            [style['title__arrow--down']]: !value,
+          },
+        )}
```

Comment:
봤을때 `--down`에 해당하는 CSS가 없다고 봣는데,, 그럼 불필요하겠죠?
arrow에 해당하는 css를 작성하고,(지금은 없다지만)
아래처럼 바꾸는 게 적절해보여요.
```suggestion
        <Arrow className={cn(
          {
            [style['title__arrow']]: true,
            [style['title__arrow--up']]: value,
          },
        )}
```

Reply:
넵 굳이 쓰이지 않는 클래스명을 만들 필요는 없었던 것 같습니다